.github/workflows/reset-db.yaml

```diff
@@ -4,6 +4,18 @@
 #     cf run-task getgov-ENVIRONMENT --command 'python manage.py flush' --name flush
 #     cf run-task getgov-ENVIRONMENT --command 'python manage.py load' --name loaddata
 
+# IMPORTANT NOTE:
+# When running this workflow, it is important to use the right source branch
+# for your target sandbox.  For example,  if you just recently pushed new migrations
```

Comment:
@CocoByte I don't think anyone will see it in the workflow yaml. So what if we added to the developer readme.md? Also, updating the dev onboarding doc (linked in the dotgov-dev slack channel) where it mentions the reset command would be great.

Reply:
Actually, I re-thought this instruction and I don't think it is correct.  This workflow should run correctly even if "main" is the source for this yaml.  I'm going to delete the instructions and keep an eye out for any other ways this might be failing.  So far I have seen zero failures for the last few months with multiple people using it.

Reply:
I've decided to look at making notes in migration-troubleshooting.md as resolution for this ticket